ansible/roles/test/tasks/pfc_wd/functional_test/storm_all_test.yml

```diff
@@ -56,6 +56,12 @@
       include_tasks: roles/test/tasks/pfc_wd/functional_test/storm_all_action.yml
       with_dict: "{{combined_dict}}"
   always:
+    - name: Stop PFC storm on fanout switch
+      action: apswitch template="{{pfc_wd_storm_stop_template}}"
+      args:
+        host: "{{peer_mgmt}}"
+        login: "{{peer_login}}"
+      connection: switch
```

Comment:
This will only work if dut is connected to a single fanout. We have duts connected to multiple fanouts, so only the current one will get cleaned up

Reply:
@neethajohn is there reference to cleanup all dut's ? I will update it accordingly . This change needed in 201811 so that also has multiple dut support ?

Reply:
Talked with Abhishek offline to explore the rescue option

Reply:
thanks @neethajohn . Updated to use rescue in storm_all_action.yml